_includes/item/metadata.html

```diff
@@ -16,9 +16,36 @@
             <a class="me-3" href="{{ t | strip | url_param_escape | prepend: '/browse.html#' | relative_url }}">{{ t | strip }}</a>
             {% endfor %}
             {% elsif f.external_link == "true" %}
-            <a href="{{ page[f.field] }}" target="_blank" rel="noopener">{{ page[f.field] }}</a>
+                {% assign links = page[f.field] | split: ";" %}
+                {% for markdown_link in links %}
+                {% if markdown_link contains "[" and markdown_link contains "]" and markdown_link contains "(" and markdown_link contains ")" %}
+                    {% assign link_text = markdown_link | split: '[' | last | split: ']' | first %}
+                    {% assign link_url = markdown_link | split: '(' | last | split: ')' | first %}
+                    {% if link_text == "" %}
```

Comment:
I don't get this. Omeka either gives you a value or a tuple of label and uri. Why do you check for empty strings?

Reply:
I thought maybe the label in omeka is not always specified, so just to catch this case. Is this not possible?

Reply:
If there is a URI than there also is a Label according to our data entry rules.